build.proj

```diff
@@ -55,7 +55,7 @@
                       Overwrite="True" />
   </Target>
 
-  <Target Name="BuildCustomTasks">
+  <Target Name="BuildCustomTasks" Condition="'$(DotNetBuildOffline)' != 'true'" >
```

Comment:
Is it safe to assume these already exist in offline mode? Should we instead do some other condition based on their existence?

Reply:
It is save to assume that they already exists.  They're being built into the Tools directory, which gets copied into the tarball.  If they don't exist, there's a problem with the original build.

Reply:
I guess it isn't clear to me that DotNetbuildOffline is only going to be used in the context of the tar-ball. I would expect it to also work on a git repo that I didn't want to restore things from the network.